palindrom-client.html

```diff
@@ -23,577 +20,449 @@
+    <style>
+        .box-container {
+            position: fixed;
+            top: 0;
+            left: 0;
+            width: 100%;
+            z-index: 1000;
+        }
+
+        .box-container[hidden] {
+            display: none;
+        }
+
+        .box {
+            padding: 5px 5px;
+        }
+
+        .notice {
+            background-color: #fff4cc;
+            border-color: #ffe070;
+            color: #000;
+        }
+
+        .error {
+            background-color: #fee;
+            border: 1px solid #f9c1c1;
+            border-top: none;
+            color: #444;
+        }
+
+        .hourglass {
+            animation: rotateHG 2s cubic-bezier(.03, 1, 1, 0) infinite;
+            transform-origin: 50% 50%;
+            display: inline-block;
+            text-decoration: none!important;
+        }
+
+        @keyframes rotateHG {
+            100% {
+                transform: rotate(360deg)
+            }
+        }
+
+        .tappable {
+            cursor: pointer;
+        }
+
+        .tappable span {
+            text-decoration: underline;
+        }
+    </style>
+    <div class="box-container reconnect-pane" hidden>
+        <div class="box notice tappable" on-tap="reconnectNow">
+            <strong>
+                <span class="hourglass">⏳</span> Not connected.
+            </strong>
+            Connecting in <span class="reconnection-seconds">5</span>s...
+            <span>Retry now</span>
+        </div>
+    </div>
+    <div class="box-container reconnecting-now-pane" hidden>
+        <div class="box notice">
+            <strong>
+                <span class="hourglass">⏳</span> Connecting now...
+            </strong>
+        </div>
+    </div>
+    <div class="box-container error-pane" hidden>
+        <div class="box error tappable">
+            <span class="hourglass">⏳</span> Connection error.</strong> See console for details.
+            <a class="reloading-in-message">
+                Reloading page in <span class="reconnection-seconds">5</span>...
+                • <span on-tap="cancelReloading">Do not reload</span> •
+            </a>
+            <span on-tap="reload">Reload now</span>
+        </div>
+    </div>
+    <div class="box-container generic-error-pane" hidden>
+        <div class="box error tappable" on-tap="reload">
+            <strong>⚠️ An error has occurred.</strong> See console for details.
+            <span>Click here to reload</span>
+        </div>
+    </div>
+</template>
+<script>      
+
+
+    (function() {
+
+        const template = document.currentScript.ownerDocument.querySelector('#palindrom-client');
+
+        const defaultValues = {
+            "useWebSocket": true,
+            "debug": false,
+            "localVersionPath": '/_ver#c$',
+            "remoteVersionPath": '/_ver#s',
+            "ot": true,
+            "purity": false,
+            "pingIntervalS": 60,
+            "path": '/',
+            "devToolsOpen": false,
```

Comment:
It does not seem to be used anywhere.

Reply:
Have we dropped this feature at all?

Reply:
Oh yes I dropeed it. Because I discovered it isn't working correctly. I mean the detection function wasn't working as it should and the binding would be much more complicated. I dropped it.

Reply:
Please just note it later in release notes